packages/core/src/abstract-dialect/query-interface.js

```diff
@@ -85,6 +85,9 @@ export class AbstractQueryInterface extends AbstractQueryInterfaceTypeScript {
     // Postgres requires special SQL commands for ENUM/ENUM[]
     await this.ensureEnums(tableName, attributes, options, model);
 
+    // Snowflake requires special SQL commands for SEQUENCES
+    await this.ensureSequences(tableName, attributes, options, model);
```

Comment:
```suggestion
    await this.ensureSequences(tableName, attributes, options);
```

Oh, and this suggestion.